### ISSUE TITLE
Only show as many E-periods as needed

### ISSUE THREAD

**Issue by RobertConde** [ufosc/Schedule_Helper#45]
<img width="573" alt="image" src="https://github.com/ufosc/SwampScheduler/assets/42223541/35c431e9-317d-45dc-8adc-68bcf8bf16b1">

Should not show E1, E2, and E3 because there are no meetings during those periods.

**Comment by IsRo06** [comment]
Hi! Would like to contribute! Is it okay if I start working on this issue?

**Comment by RobertConde** [comment]
> Hi! Would like to contribute! Is it okay if I start working on this issue?

Yeah, absolutely! I'll be at casual coding today so just let me know if you need any help.

**Comment by IsRo06** [comment]
Hey! I tried to start working on this but I'm getting a CORS API key is missing error. I'm guessing I'm missing some sort of .env file! Anyway I can get it?

**Comment by RobertConde** [comment]
Did you add the `.env` pinned in the Discord channel to the `app/` folder?

**Comment by IsRo06** [comment]
I just did! I'm not in the discord so I was a bit lost for a second. All fixed now

**Comment by IsRo06** [comment]
Hi! So I've working on this issue fora couple of days now and haven't been able to get it work properly. While I was able to get it to show no E periods at all as soon as there is a class in one of them things start getting a bit crazy (It no longer generates more schedules and the positionin gets weird)

I feel that I've been stucked bc its been really hard for me to understand how the code is written and what triggers what.


![image](https://github.com/ufosc/SwampScheduler/assets/115611774/7444951d-9373-479b-900c-c7611181d481)
 Attatched is an image of where I'm stucked rn.
 
 Bc of this i'll be taking myself off of the issue so that someone else could work on it. Sorry :(

**Comment by Nihar3430** [comment]
I'll claim this issue
